### ISSUE TITLE
Disable/enable abd 

### ISSUE THREAD

**Issue by ipozow** [madebyfusion/antibotdeluxe#11]
It would be great a feature where the bungee admins have access to disable/enable ABD. 
I got this idea to prevent false positives

**Comment by madebyfusion** [comment]
In the case of false positives you can use the analysis ID a player gets to whitelist him with one click. Just run /ab check <ID> and click whitelist. I think that's safer than disabling ABD altogether.

**Comment by ipozow** [comment]
yes but players aren't smart enough, I lost like 200 players a day per like a week

**Comment by madebyfusion** [comment]
If proxydetection is false flagging a lot of players in your region, you should simply set the check to only enable if an attack occurs. I believe that's what you are trying to achieve?
